templates/includes/sidebar.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>

```

Comment:
The `<!DOCTYPE html>`, `<html>`, `<head>`, and `<body>` tags are not necessary in this file. These tags are typically included in the base template file. Here, you should only include the specific content of the sidebar.